iocage/cli/list.py

```diff
@@ -49,9 +46,9 @@ def list_cmd(dataset_type, header, _long, remote, http, plugins):
         if not header:
             if dataset_type == "base":
                 for item in _list:
-                    lgr.info(item)
+                    print("%s" % (item))
```

Comment:
Why the string formatting? Will `print(item)` not work? Otherwise can we use `print(f"{item}")` instead.